### ISSUE TITLE
upcoming events

### ISSUE THREAD

**Issue by decker78** [digitalcolony/scrape-meetup#10]
Wow nice work.being done here. I also have a small local group and the API is too expensive to use in order to export upcoming events to for example wordpress.

I tried a lot with bs4/Python but scraping upcoming events seems not so easy.

Did you already had a look into scrapinflg upcoming events?

**Comment by digitalcolony** [comment]
No, I haven't. I'm sorry I just saw your message. GitHub never alerted me.
